commands/apps/open.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+let co       = require('co');
+let cli      = require('heroku-cli-util');
+let extend   = require('util')._extend;
+
+function* run (context, heroku) {
+  let app = context.args.app || context.app;
+  if (!app) throw new Error('No app specified');
+  app = yield heroku.apps(app).info();
+  yield cli.open(app.web_url);
+}
+
+let cmd = {
+  topic: 'apps',
+  command: 'open',
+  description: 'open the app in a web browser',
+  wantsApp: true,
+  needsAuth: true,
+  args:  [{name: 'app', hidden: true, optional: true}],
```

Comment:
@dickeyxxx This changes the undocumented behavior of the old command.  In the ruby command if there is an arg passed.
```
heroku apps:open /foo --app thawing-temple-8799
```
Opens the url `https://thawing-temple-8799.herokuapp.com//foo` rather than the app `/foo`